lib/license_finder/package_managers/bundler_package.rb

```diff
@@ -8,6 +8,7 @@ def initialize(spec, bundler_def, options={})
         spec.name,
         spec.version.to_s,
         options.merge(
+          authors: spec.authors.to_s.tr('\/[\/]', ''),
```

Comment:
In Gem::Specification, authors is an array, so why not `spec.authors.join(', ')` or something like that?